script.js

```diff
@@ -0,0 +1,24 @@
+async function fetchTodo() {
+    try {
+        const response = await fetch('https://jsonplaceholder.typicode.com/todos');
+        if (!response.ok) {
+            throw new Error('Network response was not ok');
+        }
+        const json = await response.json();
+        json.forEach((item) => appendData(item.title));
```

Comment:
`fetchTodo` is only responsible for fetching data.

`appendData` should call within window.onload

Reply:
fixed